docs/introduction/background.md

```diff
@@ -4,28 +4,36 @@ Background
 
 ![liquid chromatography setup](../images/introduction/introduction_LC.png)
 
 ## Mass Spectrometry
 
 Mass spectrometry is an analytical technique used to determine the abundance of molecules in a sample. There are three major components in a mass spectrometer:
-- An **ion source**, which generates ions from the incoming sample.
-- A **mass analyzer**, which separates the ions according to their mass-to-charge ratio. There are several types of mass analyzers. Depending on the mass analyzer used in the experiment, OpenMS offers calibration tools, so that highly accurate results can be achieved.
-- A **detector**, which records the mass-to-charge ratio of each ion against their relative abundance in a plot known as a {term}`mass spectrum`.
+- An **ion source**, which generates ions from the incoming sample. In LC-MS, the sample elutes from the LC column at different retentions times.
+- A **mass analyzer**, which separates the ions according to their mass-to-charge ratio. There are several types such as time of flight (TOF) and quadrupole mass analyzers. Depending on the mass analyzer, OpenMS offers calibration tools, so that highly accurate results can be achieved.
+- A **detector**, which records the mass-to-charge ratio of each ion against their relative abundance in a plot known as a mass spectrum.
 
 ![mass spectrometry](../images/introduction/introduction_MS.png)
 
+Sometimes a specialized form of mass spectrometry called Tandem Mass Spectrometry ({term}`MS/MS` spectrometry) is used when better identification is required. With MS/MS spectrometry,  an ion is isolated, fragmented using an inert gas and a second mass spectrum is recorded from the ion fragments. In this context, the primary ion is called the precursor ion and the spectrum from the fragments is called an  MS2 ({term}`MS/MS`) spectrum, which can help to resolve the ambiguities in identification of the precursor ion. Tandem mass spectrometry is especially useful for linear polymers like proteins, RNA and DNA and the fragments typically break the polymer into two parts. For example, peptides (short strands of amino acids, part of a protein) typically break between each of the amino acids, leading to a so-called ion ladder where the distance between each peak in the MS2 spectrum reveals the identity of the amino acid, as most amino acids have different masses. 
```

Comment:
Also, metabolites can be identified by MS2 spectrum similarity matches with annotated compounds in libraries.